### ISSUE TITLE
Error: Cannot find module 'update-notifier'

### ISSUE THREAD

**Issue by bita-c** [T1st3/generator-composer#2]
First time trying yo and generator-composer, I did the `npm install -g yo generator-composer`, and then running `yo composer`, I get:
```
module.js:340
    throw err;
    ^
Error: Cannot find module 'update-notifier'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at /usr/local/lib/node_modules/generator-composer/app/index.js:30:25
    at /usr/local/lib/node_modules/generator-composer/node_modules/figlet/lib/figlet.js:738:13
    at /usr/local/lib/node_modules/generator-composer/node_modules/figlet/lib/node-figlet.js:24:13
    at Function.me.parseFont (/usr/local/lib/node_modules/generator-composer/node_modules/figlet/lib/figlet.js:906:17)
    at /usr/local/lib/node_modules/generator-composer/node_modules/figlet/lib/node-figlet.js:23:16
    at fs.js:271:14
```
Seems like it's missing a dependency on update-notifier

**Comment by jamjon3** [comment]
Same problem. I've looked around to no avail. I tried installing the 'update-notifier' and now I get this:

```
  _   _  ___     ___ ___  _ __ ___  _ __   ___  ___  ___ _ __ 
 | | | |/ _ \   / __/ _ \| '_ ` _ \| '_ \ / _ \/ __|/ _ \ '__|
 | |_| | (_) | | (_| (_) | | | | | | |_) | (_) \__ \  __/ |   
  \__, |\___/   \___\___/|_| |_| |_| .__/ \___/|___/\___|_|   
  |___/                            |_|                        

/usr/local/lib/node_modules/update-notifier/index.js:23
		throw new Error('pkg.name and pkg.version required');
		      ^
Error: pkg.name and pkg.version required
    at new UpdateNotifier (/usr/local/lib/node_modules/update-notifier/index.js:23:9)
    at module.exports (/usr/local/lib/node_modules/update-notifier/index.js:126:23)
    at /usr/local/lib/node_modules/generator-composer/app/index.js:31:19
    at /usr/local/lib/node_modules/generator-composer/node_modules/figlet/lib/figlet.js:738:13
    at /usr/local/lib/node_modules/generator-composer/node_modules/figlet/lib/node-figlet.js:24:13
    at Function.me.parseFont (/usr/local/lib/node_modules/generator-composer/node_modules/figlet/lib/figlet.js:906:17)
    at /usr/local/lib/node_modules/generator-composer/node_modules/figlet/lib/node-figlet.js:23:16
    at fs.js:266:14
    at Object.oncomplete (fs.js:107:15)
```

I am wondering if this may be related to out of date dependencies (aka: this package is relying on an old version of 'update-notifier' which it isn't pulling down itself)

**Comment by T1st3** [comment]
hi there, sorry for this late answer.
I just updated the generator and its dependencies, and I think that it will solve the problem you both guys have.
If you can test this new version (0.12.0) of the generator, can you please confirm the issue can be closed (or not)

**Comment by jaceju** [comment]
I get same problem with version 0.12.0.

**Comment by natsu90** [comment]
Check out PR #3.

**Comment by T1st3** [comment]
Merged PR #3 about a minute ago. Tested it at home, and it seems to solve this issue.

**Comment by mahapatrakamlesh** [comment]
Kindly go to the Path :/usr/local/lib/node_modules/  and delete all the other custom generators.
